pkgs/applications/window-managers/i3/i3ipc-autotiling.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "i3ipc-autotiling";
+  version = "d521cd6d26d0b3aadbd283fe4165235b5df386f2";
+  src = fetchTarball {
+    name = pname;
+    url = "https://github.com/nwg-piotr/autotiling/archive/${version}.tar.gz";
+     # Hash obtained using `nix-prefetch-url --unpack <url>`
+     sha256 = "1lz1fasm4586vw9y0hdl2fn56swb7jbsgi71n1jwdfl3bnhawv0s";
+   };
+   propagatedBuildInputs = [ python3Packages.i3ipc ];
+
+   dontBuild = true;
+   doCheck = false;
+   installPhase = ''
+    mkdir -p "$out/bin"
+    cp autotiling.py "$out/bin/autotiling"'';
+   postInstall = ''
+    makeWrapper ${python3Packages.python.interpreter} $out/bin/${pname}-python-interpreter
```

Comment:
i3ipc-autotiling-python-interpreter is a strange naming for the executable. Also are you sure the wrapping does not happen automatically by the magic of buildPythonPackage ?

Reply:
should i fix it with commits here or should i pulled new again?

Reply:
You can do the modifications locally wit git commit --amend or git rebase -i and then `git push --force-with-lease`. This will update the content of the pull request.